blog/_src/posts/2020-10-15-shallow-mailing-list.md

```diff
@@ -0,0 +1,414 @@
+    Title: Transient Answers Old Questions
+    Date: 2020-10-15T13:32:12
+    Tags: typed racket, by Ben Greenman
+
+Several old questions from the Typed Racket mailing list have new and simple
```

Comment:
Maybe here (or elsewhere) it’s a good time to establish a connection between transient and shallow. Otherwise you just switch one for another suddenly.